mono/mini/aot-compiler.c

```diff
@@ -9354,6 +9370,8 @@ init_aot_file_info (MonoAotCompile *acfg, MonoAotFileInfo *info)
 	info->nshared_got_entries = acfg->nshared_got_entries;
 	for (i = 0; i < MONO_AOT_TRAMP_NUM; ++i)
 		info->tramp_page_code_offsets [i] = acfg->tramp_page_code_offsets [i];
+
+	generate_aotid(&info->aotid);
```

Comment:
Oh, you are right. I had code to store/load this and had this working, somehow it got lost maybe on a bad git reset. I will write/add it again.